src/fireedge/src/server/routes/api/auth/functions.js

```diff
@@ -14,566 +14,188 @@
  * limitations under the License.                                            *
  * ------------------------------------------------------------------------- */
 
-const { DateTime } = require('luxon')
-// eslint-disable-next-line node/no-deprecated-api
-const { parse } = require('url')
-const { global, Array } = require('window-or-global')
-const { Actions: ActionUsers } = require('server/utils/constants/commands/user')
-const { Actions: ActionZones } = require('server/utils/constants/commands/zone')
+const { Map } = require('immutable')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected that you are importing a module inside a function, which is known as lazy loading. If modules are imported within a function, it might prevent other requests from being handled at a more critical time. We recommend that you load all modules at the beginning of each file, before and outside of any functions.